src/subarrayn.jl

```diff
@@ -0,0 +1,25 @@
+const unsafe_getindex = Base.unsafe_getindex
+
+function Base.isnull{T,N,P<:NullableArray,IV,LD}(V::SubArray{T,N,P,IV,LD}, I::Int...)
+    @Base._inline_meta
+    @boundscheck checkbounds(V, I...)
+    @inbounds r = V.parent.isnull[Base.reindex(V, V.indexes, Base.to_indexes(I...))...]
```

Comment:
Turns out, to_indexes for some reason is necessary, if I don't use it, it gets an error, saying that it can't find a method of reindex that matches.  Any idea why?